docker/cluster/master_1/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM redis-py-base:latest
+
+COPY redis.conf /usr/local/etc/redis/redis.conf
+COPY nodes.conf /usr/local/etc/redis/nodes.conf
+
+EXPOSE 7000
```

Comment:
You'll need to use the same port changes here.

Reply:
done